golangci-lint: fix issues for containedctx and turn on

Fix

```
        internal/docker/environment.go:37:2: found a struct that contains a context.Context field (containedctx)
        	ctx context.Context
        	^
```

Add `containedctx` to enable list in `.golangci.yaml`.

--- a/.golangci.yaml
+++ b/.golangci.yaml
@@ -20,10 +20,10 @@ linters:
   # See https://golangci-lint.run/usage/linters for details.
   default: none
   enable:
+    - containedctx
     - fatcontext
     - govet
     - ineffassign
     - staticcheck
     - unparam
     - unused
-    - containedctx

--- a/.golangci.yaml
+++ b/.golangci.yaml
@@ -26,3 +26,4 @@ linters:
     - staticcheck
     - unparam
     - unused
+    - containedctx

